src/sass/_common/common.scss

```diff
@@ -728,14 +728,11 @@ label + input {
 }
 
 @media (max-width: 768px) {
-    .show_mobile {
```

Comment:
we are using this class in other files, if you want to update it we should update it everywhere